src/Components/CreateImageWizardV2/EditImageWizard.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useEffect } from 'react';
+
+import { useNavigate } from 'react-router-dom';
+
+import CreateImageWizard from './CreateImageWizard';
+import { mapRequestToState } from './utilities/requestMapper';
+
+import { useAppDispatch } from '../../store/hooks';
+import { useGetBlueprintQuery } from '../../store/imageBuilderApi';
+import { loadWizardState, initializeWizard } from '../../store/wizardSlice';
+import { resolveRelPath } from '../../Utilities/path';
+
+type EditImageWizardProps = {
+  blueprintId: string;
+};
+
+const EditImageWizard = ({ blueprintId }: EditImageWizardProps) => {
+  const dispatch = useAppDispatch();
+  const navigate = useNavigate();
+
+  const { data: blueprintDetails, error } = useGetBlueprintQuery({
+    id: blueprintId,
+  });
+  useEffect(() => {
+    if (blueprintId && blueprintDetails) {
+      const editBlueprintState = mapRequestToState(blueprintDetails);
+      dispatch(loadWizardState(editBlueprintState));
+    } else {
+      dispatch(initializeWizard());
```

Comment:
Wouldn't this mean that the wizard state was incorrect for a split second, before the useGetBlueprintQuery response was received? Would it be better to use a loading state?